src/misosoupy/psychopy_present_item_list.py

```diff
@@ -8,34 +8,35 @@
 
 import time
 
+import os
+
 # --- Import packages ---
 from psychopy import core, event, logging, visual
 
+# Import config file and screen parameters
+import psychopy_exit_out
+import setup_misosoupy
+config_path = setup_misosoupy.get_home_dir() + os.sep + 'config.ini'
```

Comment:
You could have a function dedicated to loading the config file to avoid repetition across python files, and in case anyone wants to change the path at some point it would be easier

Reply:
ooh okay yes, do you think I should just store the config file in the assets directory, so the path gets loaded in the get_path_to_assets function? Or make a get_path_to_config function with similar code?

Or, alternatively, maybe I just define the path to the config file directly in setup_misosoupy, and remove it as a necessary input when calling the function? So that way I wouldn't need to redefine the path across python files?

Reply:
I think these are all options! Even if you move it to the assets directory I would still either create a function like get_path_to_config or defining it as a constant in setup_misosoupy (you can then import that variable).